lbrynet/core/Wallet.py

```diff
@@ -1,1325 +0,0 @@
-                if self._is_first_run():
-                    log.info("Running the wallet for the first time. This may take a moment.")
-                    self.printed_retrieving_headers = True
-                return False
-            self._start_check.stop()
-            self._start_check = None
-            if self.network.is_connected():
-                network_start_d.callback(True)
-            else:
-                network_start_d.errback(ValueError("Failed to connect to network."))
-
-        self._start_check = task.LoopingCall(check_started)
-
-        d = setup_network()
-        d.addCallback(lambda _: self._load_wallet())
-        d.addCallback(lambda _: self._start_check.start(.1))
-        d.addCallback(lambda _: network_start_d)
-        d.addCallback(lambda _: self._load_blockchain())
-        d.addCallback(lambda _: log.info("Subscribing to addresses"))
-        d.addCallback(lambda _: self.wallet.wait_until_synchronized(lambda _: None))
-        d.addCallback(lambda _: log.info("Synchronized wallet"))
-        d.addCallback(lambda _: self.get_cmd_runner())
-        d.addCallbacks(lambda _: log.info("Set up lbryum command runner"))
-        return d
-
-    def _stop(self):
-        if self._start_check is not None:
-            self._start_check.stop()
-            self._start_check = None
-
-        if self._catch_up_check is not None:
-            if self._catch_up_check.running:
-                self._catch_up_check.stop()
-            self._catch_up_check = None
-
-        d = defer.Deferred()
-
-        def check_stopped():
-            if self.network:
-                if self.network.is_connected():
-                    return False
-            stop_check.stop()
-            self.network = None
-            d.callback(True)
-
-        if self.wallet:
-            self.wallet.stop_threads()
-            log.info("Stopped wallet")
-        if self.network:
-            self.network.stop()
-            log.info("Stopped connection to lbryum server")
-
-        stop_check = task.LoopingCall(check_stopped)
-        stop_check.start(.1)
-        return d
-
-    def _load_wallet(self):
-        path = self.config.get_wallet_path()
-        storage = lbryum_wallet.WalletStorage(path)
-        wallet = lbryum_wallet.Wallet(storage)
-        if not storage.file_exists:
-            self.is_first_run = True
-            seed = wallet.make_seed()
-            wallet.add_seed(seed, None)
-            wallet.create_master_keys(None)
-            wallet.create_main_account()
-            wallet.synchronize()
-        self.wallet = wallet
-        self.is_wallet_unlocked = not self.wallet.use_encryption
-        self._check_large_wallet()
-        return defer.succeed(True)
-
-    def _check_large_wallet(self):
-        addr_count = len(self.wallet.addresses(include_change=False))
-        if addr_count > 1000:
-            log.warning("Your wallet is excessively large (%i addresses), "
```

Comment:
Other than the wallet being slow at this point (via implementation), what's the harm?  If the wallet is used, it's inevitably going to reach this point.

Reply:
You are commenting on a file that was deleted.